src/app/components/primary-navbar/primary-navbar.component.html

```diff
@@ -1,10 +1,12 @@
 <ng-template #desktop>
     <div class="desktop-navbar" *ngIf="showNav">
         <div class="nav-wrapper">
-            <div class="logo" [routerLink]="['/home']">
-                <span class="logo-image"></span>
-                <span id="clark-logo" class="logo-link">CLARK</span>
-            </div>
+            <a [routerLink]="['/home']" target="_blank" style="text-decoration: none;">    
```

Comment:
Just curious as to why we're opening the link in a new tab. The traditional user flow is to open in the same tab, and I'm just curious as to the reason for the deviation.

Reply:
If you take a glance at the story, it asks to make the logo open up the homepage in a new tab. Not sure the reason for the deviation but those were the directions

Reply:
@katieblomquist good catch! The target should be removed for the home button. Users should remain within CLARK when clicking the logo

Reply:
Gotchya, I'll get that story revised too so there's no confusion